src/fundus/publishers/us/washington_post.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class WashingtonPostParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div[data-qa='article-body'] > p, div[class='story relative'] > p")
+        _subheadline_selector = CSSSelector("h2[data-qa='subheadline']")
```

Comment:
As it seems the name is a bit misleading. What you have selected now is what we would call the summary within Fundus. The subheadline (for us) is something like the `He can't believe it` in this article https://www.washingtonpost.com/elections/2024/04/27/trump-trial-election-2024-campaign/